net/golioth/CMakeLists.txt

```diff
@@ -12,23 +12,20 @@ zephyr_library_sources_ifdef(CONFIG_GOLIOTH_SETTINGS settings.c)
 zephyr_library_sources_ifdef(CONFIG_GOLIOTH_SYSTEM_CLIENT system_client.c)
 
 if(CONFIG_GOLIOTH_AUTH_METHOD_CERT)
-  foreach(type IN ITEMS ca crt key)
-    string(TOUPPER ${type} type_upper)
-    set(path ${CONFIG_GOLIOTH_SYSTEM_CLIENT_${type_upper}_PATH})
+  set(path ${CONFIG_GOLIOTH_SYSTEM_CLIENT_CA_PATH})
```

Comment:
The CA cert stays hardcoded in the Golioth library for now because it is the same across the entire fleet. Eventually, we'll need a way to provision this certificate, but that is part of a larger story around rolling the CA certificate that is out of scope of these changes.